rewrite.yml

```diff
@@ -0,0 +1,41 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: DefinitionOfDone
+displayName: Definition of Done
+description: Automatically cleanup code to comply checkstyle convention.
+recipeList:
+  # - org.openrewrite.java.migrate.UpgradeToJava17
+  # - org.openrewrite.java.recipes.RecipeNullabilityBestPractices
+  # - org.openrewrite.java.recipes.RecipeTestingBestPractices
+  # - org.openrewrite.maven.BestPractices
```

Comment:
39m 15s

- https://app.circleci.com/pipelines/github/checkstyle/checkstyle/36307/workflows/00fb2683-4952-4399-9fe7-1be9e358fbc8/jobs/1057259

we might need longer as the refasterrules are composite recipes covering a lot. I would suggest to start and then see what recipe is worth the effort.

@romani

Reply:
should be happy ci now. Thanks for considering. This will improve quality and dev experience.